### PR TITLE
Add ISO8583-connector-example.md

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+    
+## Testing
+
+1. Login in to the Micro Integrator CLI tool.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]:
```suggestion
1. Log in to the Micro Integrator CLI tool.
```

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+| serverHost   | The host of the server. |
+| serverPort   | The port on which the server can start listening for messages. |
+
+**Sending an ISO8583 message**
+
+To send an ISO8583 message, you need to use the <iso8583.sendMessage/> operation in your configuration. You can use a REST client to send an XML message, set the Content-Type header in the REST client as application/xml, and then POST the body in XML format.
+
+The following diagram illustrates how to send an ISO8583 message using a REST client.
+
+<img src="/assets/img/connectors/ISO8583-connector.png" title="ISO8583 Connector" width="800" alt="ISO8583 Connector"/>
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Shall we add some description of what exactly is happening in the diagram?

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+## What you'll build
+
+There are many such standards exist. The ISO8583 message format is used for financial transaction, such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce etc. transactions.
+The financial transaction is communication between two systems through a socket connection. After connection established, each system can send messages in ISO 8583 format which commonly will be requested and the other system will response. 
+
+For an example the purchase made in a store may travel from the merchant terminal through another require a network or networks to the issuing bank where the card holders account is held.
+Cardholder-originated transactions include purchase, withdrawal, deposit, refund, reversal, balance inquiry, payments and inter-account transfers. ISO 8583 also defines system-to-system messages for secure key exchanges, reconciliation of totals, and other administrative purposes.The response or authorizing or decline the transaction needs to be returned by the same route to the terminal.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]:
```suggestion
Cardholder-originated transactions include purchase, withdrawal, deposit, refund, reversal, balance inquiry, payments, and inter-account transfers. ISO8583 also defines system-to-system messages for secure key exchanges, reconciliation of totals, and other administrative purposes. The response on authorizing or declining the transaction needs to be returned by the same route to the terminal.
```

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
+# ISO8583 Connector Example
+
+Given below is a sample scenario that demonstrates how to WSO2 ISO8583 connector send an ISO8583 message to the financial networks using WSO2 micro integrator.
+
+## What you'll build
+
+There are many such standards exist. The ISO8583 message format is used for financial transaction, such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce etc. transactions.
+The financial transaction is communication between two systems through a socket connection. After connection established, each system can send messages in ISO 8583 format which commonly will be requested and the other system will response. 
+
+For an example the purchase made in a store may travel from the merchant terminal through another require a network or networks to the issuing bank where the card holders account is held.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]:
```suggestion
For an example the purchase made in a store may travel from the merchant terminal through another. This requires a network or networks to the issuing bank where the card holder's account is held.
```

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
+# ISO8583 Connector Example
+
+Given below is a sample scenario that demonstrates how to WSO2 ISO8583 connector send an ISO8583 message to the financial networks using WSO2 micro integrator.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]:
```suggestion
Given below is a sample scenario that demonstrates how the WSO2 ISO8583 Connector sends an ISO8583 message to financial networks using WSO2 Micro Integrator.
```

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+## What you'll build
+
+There are many such standards exist. The ISO8583 message format is used for financial transaction, such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce etc. transactions.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]:
```suggestion
The ISO8583 message format is used for financial transactions such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce, etc. transactions.
```

Removed the first line. I think it's not required.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+The financial transaction is communication between two systems through a socket connection. After connection established, each system can send messages in ISO 8583 format which commonly will be requested and the other system will response. 
+
+For an example the purchase made in a store may travel from the merchant terminal through another require a network or networks to the issuing bank where the card holders account is held.
+Cardholder-originated transactions include purchase, withdrawal, deposit, refund, reversal, balance inquiry, payments and inter-account transfers. ISO 8583 also defines system-to-system messages for secure key exchanges, reconciliation of totals, and other administrative purposes.The response or authorizing or decline the transaction needs to be returned by the same route to the terminal.
+
+Given below is a sample API that illustrates how you can configure with the `init` operation and then use the `iso8583.sendMessage` operation for the send an ISO8583 message to the financial transactions.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]:
```suggestion
Given below is a sample API that illustrates how you can configure ISO8583 with the `init` operation and then use the `iso8583.sendMessage` operation to send an ISO8583 message for the financial transactions.
```

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+               <log/>
+               <send/>
+           </outSequence>
+           <faultSequence/>
+       </resource>
+   </api>
+        ```
+2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+
+3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
```
[the remark this answers is not in the document]
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md (hunk 5 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md (hunk 6 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
+# ISO8583 Connector Example
+
+Given below is a sample scenario that demonstrates how to WSO2 ISO8583 connector send an ISO8583 message to the financial networks using WSO2 micro integrator.
+
+## What you'll build
+
+There are many such standards exist. The ISO8583 message format is used for financial transaction, such as ATM, POS, Credit Card, Mobile Banking, Internet Banking, KIOSK, e-commerce etc. transactions.
+The financial transaction is communication between two systems through a socket connection. After connection established, each system can send messages in ISO 8583 format which commonly will be requested and the other system will response.
```
[the remark this answers is not in the document]
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md (hunk 7 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-example.md]: Added.

**Pull request merged**